pkg/storaged/client.js

```diff
@@ -1354,6 +1354,7 @@ function legacy_vdo_overlay() {
     function start() {
         let buf = "";
 
+        // legacy vdo monitoring (RHEL-8)
```

Comment:
The whole function is called `legacy_vdo_overlay()` with a comment re RHEL 8 at the top, I don't think we need this here in addition... or?

Reply:
Okido re-pushed